components/TopBarMobileMenu.js

```diff
@@ -51,34 +51,26 @@ class TopBarMobileMenu extends React.Component {
         data-cy="user-menu"
       >
         <Box as="ul" my={2} pl={0}>
-          {menuItems.discover && (
-            <ListItem>
-              <Link href="/discover" onClick={closeMenu}>
-                <FormattedMessage id="menu.discover" defaultMessage="Discover" />
-              </Link>
-            </ListItem>
-          )}
-          {menuItems.howItWorks && (
-            <ListItem>
-              <Link href="/how-it-works" onClick={closeMenu}>
-                <FormattedMessage id="menu.howItWorks" defaultMessage="How it Works" />
-              </Link>
-            </ListItem>
-          )}
-          {menuItems.pricing && (
-            <ListItem>
-              <Link href="/pricing" onClick={closeMenu}>
-                <FormattedMessage id="menu.pricing" defaultMessage="Pricing" />
-              </Link>
-            </ListItem>
-          )}
-          {menuItems.docs && (
-            <ListItem>
-              <a href="/help" onClick={closeMenu}>
-                <FormattedMessage id="menu.docs" defaultMessage="Docs & Help" />
-              </a>
-            </ListItem>
-          )}
+          <ListItem>
```

Comment:
Sounds like changes on this component will impact everyone regardless of the feature flag, are we ok with that?

Reply:
Ops, this is something I forgot to revert. I've extracted the new developments to `TopBarMobileMenuV2.js`.